lib/config/presets/index.spec.ts

```diff
@@ -11,6 +12,11 @@ import {
   PRESET_RENOVATE_CONFIG_NOT_FOUND,
 } from './util';
 import * as presets from '.';
+import {
+  isPresetForShallowConfig,
+  logShallowConfig,
+  updateShallowConfig,
+} from '.';
```

Comment:
are the imports already covered by 1 line above ?

Reply:
yes